src/plugins/kibana_usage_collection/server/plugin.ts

```diff
@@ -93,11 +95,17 @@ export class KibanaUsageCollectionPlugin implements Plugin {
     this.uiSettingsClient = uiSettings.asScopedToClient(savedObjectsClient);
     core.metrics.getOpsMetrics$().subscribe(this.metric$);
     this.coreUsageData = core.coreUsageData;
-    startTrackingEventLoopDelaysUsage(this.savedObjectsClient, this.pluginStop$.asObservable());
+    startTrackingEventLoopDelaysUsage(
+      this.savedObjectsClient,
+      this.instanceUuid,
+      this.pluginStop$.asObservable(),
+      new EventLoopDelaysMonitor()
+    );
     startTrackingEventLoopDelaysThreshold(
       this.eventLoopUsageCounter,
       this.logger,
-      this.pluginStop$.asObservable()
+      this.pluginStop$.asObservable(),
+      new EventLoopDelaysMonitor()
```

Comment:
Do we need distinct instances of `EventLoopDelaysMonitor` here?

Reply:
Yes as they are reset on different cadence